backend/app/api/endpoints/ai_assist.py

```
@@ -0,0 +1,308 @@
+        model_spec=model_spec,
+        user_id=user.id,
+        user_name=user.user_name or "",
+    )
+
+    return model_config
+
+
+@router.post("/process")
+async def process_ai_assist(
+    request: AIAssistRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)